examples/lib/defaultHtmlStepUi.js

```diff
@@ -273,6 +275,25 @@ function DefaultHtmlStepUi(_sequencer, options) {
     }
   }
 
+  function imageHover(step){
+
+    var img = $(step.imgElement);
+
+    img.mousemove(function(e) { 
```

Comment:
I'm a bit confused. Why is the mousemove event handler added again here if it is already added above?